salt/modules/chocolatey.py

```diff
@@ -836,39 +871,21 @@ def version(name, check_remote=False, source=None, pre_versions=False):
         salt "*" chocolatey.version <package name>
         salt "*" chocolatey.version <package name> check_remote=True
     '''
-    choc_path = _find_chocolatey(__context__, __salt__)
-    if not choc_path:
-        err = 'Chocolatey not installed. Use chocolatey.bootstrap to install the Chocolatey package manager.'
-        log.error(err)
-        raise CommandExecutionError(err)
+    installed = list_(narrow=name, local_only=True)
 
-    cmd = [choc_path, 'list', name]
-    if not salt.utils.is_true(check_remote):
-        cmd.append('-localonly')
-    if salt.utils.is_true(pre_versions):
-        cmd.append('-prerelease')
-    if source:
-        cmd.extend(['-source', source])
+    packages = {}
+    for pkg in installed:
+        if name.lower() in pkg.lower():
```

Comment:
It should not be necessary to `lower()` the name each time, since it is not being iterated upon in this loop, right?